features/strict-keywords.feature

```diff
@@ -0,0 +1,67 @@
+Feature: Require that Gherkin keyword matches to invoke step definition
+  In order to ensure I am using the correct step definition
+  As a Behat developer
+  The test should fail when I use the wrong Gherkin keyword to invoke a step definition
+
+  Background:
+    Given a context file "features/bootstrap/FeatureContext.php" containing:
+        """
+        <?php
+
+        use Behat\Behat\Context\Context;
+
+        /**
+         * Context for definition keywords.
+         */
+        class FeatureContext implements Context
+        {
+            /** @Given the system has some state */
+            public function ensureSomething(): void {}
+
+            /** @When I perform some action */
+            public function doSomething(): void {}
+
+            /** @Then I assert something */
+            public function assertSomething(): void { }
+        }
+        """
+
+  Scenario: Scenario fails if wrong keyword is used when strict keywords is enabled
```

Comment:
do we have tests for happy paths?? Should we also have one with multiple `And`s??